.github/dependabot.yml

```diff
@@ -4,3 +4,47 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "cargo"
```

Comment:
Might it be possible to streamline this so that we do not have to touch this file whenever we add another example? I think this will likely be forgotten.

Reply:
See my description of this PR, wildcards are not yet possible.

Reply:
I'll create a follow-up issue so that we don't forget to migrate once wildcards are supported.